build/clean-locks.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Utility for cleaning package locks.
+ * Usage: node build/clean-locks.js
+ */
+
+var glob = require('glob');
+var rimraf = require('rimraf');
+
+glob('./packages/**/package-lock.json', (err, files) => {
```

Comment:
you'll need to enumerate `glob` as a dependency - or, probably a better route - `rimraf` actually supports globbing, so you may be able to omit that dependency and change this code to just `rimraf('./packages/**/package-lock.json', (err) => { ... })`